Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:18
+
+WORKDIR /test-suite
+
+RUN apt update
+RUN apt install default-jre -y
```

Comment:
it's a dependency for the allure reporter, not required but having it will allow the image container to serve an allure report natively. More of a nice to have.

Reply:
I've added a comment to reflect this